dev/releases/make_archives.py

```diff
@@ -130,37 +141,40 @@
     # now create the file package-infos.json
     # We first build the json package, then create the package-infos.json, then
     # clean up the json package again.
+    notice("Compiling json package")
     path_to_json_package = glob.glob(f'{tmpdir}/{basename}/pkg/json*')[0]
     with working_directory(path_to_json_package):
         subprocess.run(["./configure"], check=True)
         subprocess.run(["make"], check=True)
 
+    notice("Constructing package_infos json file")
     package_infos = subprocess.run(["./bin/gap.sh", "-r", "--quiet", "--quitonbreak",
                                     "dev/releases/PackageInfos-to-JSON.g"],
                                     check=True, capture_output=True, text=True)
-    package_infos = package_infos.stdout
-
     with working_directory(tmpdir):
         with gzip.open("package-infos.json.gz", 'wb') as file:
-            file.write(package_infos.encode('utf-8'))
+            file.write(package_infos.stdout.encode('utf-8'))
+        manifest_list.append("package-infos.json.gz")
 
+    notice("Cleaning up the json package")
     with working_directory(path_to_json_package):
         subprocess.run(["make", "clean"], check=True)
-        subprocess.run(["rm", "-rf", "bin/", "Makefile"],
-                       check=True)
+        subprocess.run(["rm", "-rf", "bin/", "Makefile"], check=True)
```

Comment:
Aside: we really should use `shutil.rmtree` or `os.removedirs` instead of calling `rm -rf` (and `os.remove` to delete `Makefile`)